pkg/polaris/aws/aws.go

```diff
@@ -78,7 +78,7 @@ func (c CloudAccount) Feature(feature core.Feature) (Feature, bool) {
 
 // Feature for Amazon Web Services accounts.
 type Feature struct {
-	Name     core.Feature
+	Feature  core.Feature
```

Comment:
Maybe we want to embed `core.Feature` instead, to avoid some of the `feature.Feature.Foo..`, e.g.:
```
type Feature struct {
    core.Feature
    Regions []string
    // ..
```